folium/features.py

```diff
@@ -241,6 +241,14 @@ def __init__(self, data, style_function=None):
             else:  # This is a filename
                 self.embed = False
                 self.data = data
+        elif hasattr(data, 'to_crs'):
```

Comment:
My goal was to avoid importing geopandas explicitely in folium.
Otherwise, we'll have another dependency.

Maybe there's some other way to filter that.

    'geopandas' in data.__class__().__str__()

is ugly, but may work.